src/firmware_new/boards/frankie_v1/udp_multicast.c

```diff
@@ -0,0 +1,123 @@
+#include "udp_multicast.h"
+
+#include "firmware_new/proto/control_fw.pb.h"
+#include "lwip/api.h"
+#include "lwip/inet.h"
+#include "lwip/ip_addr.h"
+#include "lwip/memp.h"
+#include "lwip/opt.h"
+#include "lwip/pbuf.h"
+#include "lwip/sys.h"
+#include "lwip/tcp.h"
+#include "lwip/udp.h"
+#include "main.h"
+#include "pb.h"
+#include "pb_decode.h"
+#include "pb_encode.h"
```

Comment:
Similar comment here, we should use absolute includes whenever possible.

Reply:
the library headers reference the files globally, so if I add `include_prefix` then the library no longer works. All the files are just global in the repo, there is no folder structure for nanopb